examples/language-modeling/run_clm.py

```diff
@@ -606,9 +606,9 @@ def compute_metrics(eval_preds):
         # Data collator will default to DataCollatorWithPadding, so we change it.
         data_collator=default_data_collator,
         compute_metrics=compute_metrics if training_args.do_eval and not is_torch_tpu_available() else None,
-        preprocess_logits_for_metrics=preprocess_logits_for_metrics
-        if training_args.do_eval and not is_torch_tpu_available()
-        else None,
+        preprocess_logits_for_metrics=(
+            preprocess_logits_for_metrics if training_args.do_eval and not is_torch_tpu_available() else None
+        ),
```

Comment:
Is it just formatting?
Anyways those example are automatically generated from the Transformers library so this change will be overriden when we synchronize with the next release.

Reply:
If I don't apply them, then the check-code-quality fails. I could exclude the examples from it though ...

Reply:
No worries, I just wanted to check if the change was only about styling. After reading the rest of the PR I understood it was automatic reformatting.